apps/E2E/src/LinkExperimental/specs/ExperimentalLink.spec.win.ts

```diff
@@ -21,6 +21,41 @@ describe('Link Testing Initialization', function () {
   });
 });
 
+describe('Link Testing Functionality', function () {
+  /* Scrolls and waits for the Button to be visible on the Test Page */
```

Comment:
Nit: Comment says "Button". In this case, we should replace with "Link component"